test/engine/TextIndexScanTestHelpers.h

```diff
@@ -49,8 +49,8 @@ inline const TextRecordIndex getTextRecordIdFromResultTable(
 inline std::string getEntityFromResultTable(const QueryExecutionContext* qec,
                                             const Result& result,
                                             const size_t& rowIndex) {
-  return qec->getIndex().indexToString(
-      result.idTable().getColumn(1)[rowIndex].getVocabIndex());
+  return std::string{qec->getIndex().indexToString(
+      result.idTable().getColumn(1)[rowIndex].getVocabIndex())};
```

Comment:
Just so I get this correctly: Depending on if the flag `QLEVER_VOCAB_UNCOMPRESSED_IN_MEMORY` is set or not, `indexToString` will return a different string-like type, and calling the constructor makes sure that it's always a string, right? Please comment, otherwise some people (most notably future me) might get the impression that this is completely redundant.